cli-wrap/src/main/resources/cli/listings.py

```diff
@@ -512,10 +512,25 @@ def prod_difference():
     
     if style == "pretty":
         if "errorType" in output:
-            print output['errorMessage']
+            print(output['errorMessage'])
             exit()
         pretty_out = pretty.difference(output)
         print(pretty_out)
         exit()
     else:
         print(json.dumps(output))
+              
+async def get_response(query):
```

Comment:
output is future object and you have to wait until it is complete -> output = asyncio.get_event_loop().run_until_complete(get_response(query))